### PR TITLE
Add an (experimental) tutorial for how to use Plans

### PR DESCRIPTION
**Pull request by LaRiffle**

### CODE REVIEW

File: syft/__init__.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 from syft.workers import VirtualWorker
 from syft.workers import Plan
 from syft.workers import func2plan
+from syft.workers import meth2plan
```
Comment (robert-wagner) [inline syft/__init__.py]: Can we change the name to not be meth2plan, more something like method2plan (certain connotations in english)

**Comment by iamtrask** [comment]
This tutorial is some of the best light vacation reading I've ever had. Really really excellent.

File: syft/__init__.py (hunk 1 above)
Comment (LaRiffle) [reply syft/__init__.py]: Hahaha yes you're right!

**Comment by LaRiffle** [comment]
> This tutorial is some of the best light vacation reading I've ever had. Really really excellent.

Thank you! :)

**Pull request merged**